src/main/java/com/synopsys/integration/alert/web/config/FieldValidationAction.java

```diff
@@ -53,7 +53,9 @@ public void validateConfig(Map<String, ConfigField> descriptorFields, FieldModel
                 fieldErrors.add(AlertFieldStatus.error(key, ConfigField.REQUIRED_FIELD_MISSING));
             }
 
-            if (!fieldErrors.contains(key) && optionalFieldValue.isPresent()) {
+            boolean containsKey = fieldErrors.stream()
+                                      .anyMatch(fieldErrorName -> key.equals(fieldErrorName.getFieldName()));
```

Comment:
Could this be changed to the following?
```suggestion
                                      .map(AlertFieldStatus::getFieldName)
                                      .anyMatch(key::equals);
```